bug-1668861-pref-measure-set-to-default-adoption-impact-of-chang-release-81-83.toml

```diff
@@ -0,0 +1,27 @@
+[experiment]
+reference_branch = "modal-default-browser-notification"
+
+[metrics]
+daily = ["was_ever_default_browser", "was_never_default_browser"]
+weekly = ["was_ever_default_browser", "was_never_default_browser", "retained", "active_hours", "uri_count", "search_count"]
+overall = ["was_ever_default_browser", "was_never_default_browser", "active_hours", "uri_count", "search_count"]
```

Comment:
`"retained", "active_hours", "uri_count", "search_count"` are included in https://github.com/mozilla/jetstream/blob/main/jetstream/config/default_metrics.toml so they'll always be reported; jetstream merges this config with the default to decide what to compute.

Reply:
Wasn't sure about whether this custom configuration will overwrite the default metrics or not. Looks like it adds to the existing ones. Neat!